src/org/labkey/test/pages/admin/RenameFolderPage.java

```diff
@@ -0,0 +1,84 @@
+package org.labkey.test.pages.admin;
+
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class RenameFolderPage extends LabKeyPage<RenameFolderPage.ElementCache>
+{
+    public RenameFolderPage(WebDriver driver)
+    {
+        super(driver);
+    }
+
+    public static ReorderFoldersPage beginAt(WebDriverWrapper driver, String containerPath)
+    {
+        driver.beginAt(WebTestHelper.buildURL("admin", containerPath, "renameFolder"));
+        return new ReorderFoldersPage(driver.getDriver());
```

Comment:
Do you need a `waitForPageLoad` or something like to make sure the page is loaded before returning?

Reply:
page is loading quickly